third_party/move/move-compiler-v2/src/plan_builder.rs

```diff
@@ -505,10 +505,17 @@ fn convert_location(env: &GlobalEnv, attr: Attribute) -> Option<ModuleId> {
     match value {
         AttributeValue::Name(id, opt_module_name, _sym) => {
             let vloc = env.get_node_loc(id);
-            convert_module_id(env, vloc, opt_module_name)
+            let module_id_opt = convert_module_id(env, vloc.clone(), opt_module_name);
+            if !_sym.display(env.symbol_pool()).to_string().is_empty() || module_id_opt.is_none() {
```

Comment:
```suggestion
            if !sym.display(env.symbol_pool()).to_string().is_empty() || module_id_opt.is_none() {
```

Reply:
Also change `_sym` -> `sym` elsewhere.

Reply:
Is there a test case corresponding to `!_sym.display(env.symbol_pool()).to_string().is_empty()`?

Reply:
yes, in the test case where we have `#[expected_failure(out_of_gas, location=0x1::m::t0)]` and `_sym` corresponds to `t0`.

Reply:
And when would it be an empty string?

Reply:
when it is a module: `0x1::m`. Added a test for this case.